test/verify/check-machines-dbus

```diff
@@ -235,6 +235,40 @@ class TestMachinesDBus(machineslib.TestMachines):
         b.click("#activate-pool-myPoolOne-{0}".format(connectionName))
         b.wait_in_text("#pool-myPoolOne-{0}-state".format(connectionName), "active")
 
+        # Backup pool XML to redefine right after
+        m.execute("virsh pool-dumpxml myPoolTwo > /tmp/myPoolTwo.xml")
+
+        b.click("#deactivate-pool-myPoolTwo-{0}".format(connectionName))
+
+        # Delete an inactive Pool. It's volumes won't be deleted
+        b.wait_present("#delete-pool-myPoolTwo-{0}".format(connectionName))
+        b.click("#delete-pool-myPoolTwo-{0}".format(connectionName))
+        b.wait_present("div.modal-dialog")
+        b.wait_in_text("div.modal-dialog div.modal-body div.ct-form-layout",  "Its content will not be deleted.")
+        b.click('div.modal-footer button:contains("Delete")')
+        b.wait_not_present("div.modal-dialog")
+        b.wait_not_present("#pool-myPoolTwo-{0}-storage-volumes-list".format(connectionName))
+        self.assertNotIn("Empty", m.execute('[ "$(ls -A /mnt/vm_two)" ] || echo Empty').strip())
```

Comment:
Ah, I didn't check the return code of ls. I put it inside $() so I checked the output of it which is empty for empty dirs otherwise some string. And inside a conditional check of [] this return 0 or 1 like this.
``` bash
$ [ 'test' ] || echo Empty
$ [ ] || echo Empty
```
But I agree your command is cleaner ;) 
Empty